en/Editing and formatting/Properties.md

```diff
@@ -3,35 +3,122 @@ aliases:
   - front matter
   - Advanced topics/YAML front matter
   - metadata
+  - property
 ---
-While most of the text in a note is meant to be read by a human, _properties_ are text that's meant to be easily readable by a program, for example a [[Community plugins|community plugin]] or Obsidian itself.
+Properties help your organize information about a note. Adding properties to a note helps you track structured data such as text, links, dates and numbers. Properties can also be used in combination with [[Community plugins]] that can do useful things with your structured data.
 
-You can add properties to your notes by adding a block on the first line of your note. The block must start and end with three hyphens (`---`).
+## Basic usage
 
-For example, the following note contains two pieces of properties, `tag` and `publish`:
+There are several ways to add a property to a note:
 
-```yaml
----
-tag: journal
-publish: false
----
+- Use the **Add file property** [[Command palette|command]].
+- Use the **`Cmd/Ctrl+;`** [[Custom hotkeys|hotkey]].
+- Choose **Add file property** from the **More actions** menu (three dots icon) or right-clicking the tab.
+- Type `---` at the very beginning of a file.
 
-# Daily note
+Once you add a property, a row will appear at the top of the file with two inputs: the property _name_ and the property _value_.
 
-Today I learned about front matter.
-```
+For the name, you can choose anything you like. Note that Obsidian provides several default properties: `tags`, `cssclasses`, and `aliases`.
 
-> [!tip]
-> By default, properties are only visible in the [[Editing and previewing Markdown#Editor views|editing view]].
->
-> To display properties in reading view:
->
-> 1. Open **Settings**.
-> 2. Under **Editor**, enable **Show frontmatter**.
+Once you choose the property name you can give it a value.
+
+### Property types
+
+In addition to a name and value, properties also have a type. A _property type_ describes the kind of values can be stored. To change the type of a property click the property's icon or use the **Edit file property** command.
```

Comment:
_click the property's_

Our style guide indicates we ought to use **Select** instead of **Click**. 

https://help.obsidian.md/Contributing+to+Obsidian/Style+guide#UI+and+interactions